apps/ui/src/components/AppLink.vue

```diff
@@ -23,10 +23,23 @@ function normalize(to: any) {
     <a
       v-bind="$attrs"
       :href="href"
-      :class="{ [`${activeClass}`]: isActive }"
+      :class="{
+        [activeClass!]: isActive && activeClass,
+        button: button
+      }"
       @click="navigate"
     >
       <slot />
     </a>
```

Comment:
There is a actual button inside `a` on snapshot.box
![image](https://github.com/user-attachments/assets/79004aba-9dd6-457f-a12c-6e2a433830f7)

but in local
![image](https://github.com/user-attachments/assets/936f5af5-b64d-497e-a7e9-6fd3e97f19cc)


I think it should be like this (also handles tab index)
```ts
    <a
      v-bind="!button ? $attrs : undefined"
      :tabindex="button ? -1 : undefined"
      :href="href"
      :class="{
        [activeClass!]: isActive && activeClass
      }"
      @click="navigate"
    >
      <button
        v-if="button"
        type="button"
        v-bind="button ? $attrs : undefined"
        :class="{ button: button }"
      >
        <slot />
      </button>
      <slot v-else />
    </a>